test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresInCompilerGeneratedCode.cs

```diff
@@ -1052,18 +1276,27 @@ static async Task<int> MethodAsync ()
 		}
 
 		[RequiresUnreferencedCode ("--MethodWithRequires--")]
+		[RequiresAssemblyFiles ("--MethodWithRequires--")]
 		static void MethodWithRequires ()
 		{
 		}
 
 		class TypeWithMethodWithRequires
 		{
 			[RequiresUnreferencedCode ("--TypeWithMethodWithRequires.MethodWithRequires--")]
+			[RequiresAssemblyFiles ("--TypeWithMethodWithRequires.MethodWithRequires--")]
 			static void MethodWithRequires ()
 			{
 			}
 		}
 
+		[RequiresUnreferencedCode ("Message from --InitString--")]
+		[RequiresAssemblyFiles ("Message from --InitString--")]
+		public static string InitString ()
```

Comment:
Can you please give this a better name? Something like `MethodWithRequiresAndReturns` or similar.